projects/1.20/assets/embeddium/sodium/lang/zh_cn.json

```
@@ -0,0 +1,67 @@
+{
+	"sodium.option_impact.low": "低",
+	"sodium.option_impact.medium": "中",
+	"sodium.option_impact.high": "高",
+	"sodium.option_impact.extreme": "极高",
+	"sodium.option_impact.varies": "视情况而定",
+	"sodium.options.pages.quality": "质量",
+	"sodium.options.pages.performance": "性能",
+	"sodium.options.pages.advanced": "高级",
+	"sodium.options.view_distance.tooltip": "控制渲染地形的距离。距离越短，渲染的地形就越少，从而提高帧率。",
+	"sodium.options.simulation_distance.tooltip": "模拟距离控制地形和实体加载和更新的距离。较短的距离可以减少内部服务器的负载，并可能提高帧率。",
+	"sodium.options.brightness.tooltip": "控制游戏的亮度（伽玛值）。",
+	"sodium.options.gui_scale.tooltip": "设置用于用户界面的最大缩放比例。 如果设置为“自动”，则将始终使用最大比例。",
+	"sodium.options.fullscreen.tooltip": "如果启用，游戏将全屏显示（如果支持）。",
+	"sodium.options.v_sync.tooltip": "如果启用，游戏的帧率将与显示器的刷新率同步，以整体输入延迟为代价获得更流畅的体验。如果系统性能太低，此设置可能会降低帧率。",
+	"sodium.options.fps_limit.tooltip": "限制每秒的最大帧数。这有助于减少多任务处理时的电池使用和系统资源负载。如果启用了垂直同步，则此选项将被忽略，除非设置的值低于显示器的刷新率。",
+	"sodium.options.view_bobbing.tooltip": "如果启用，玩家的视野将在移动时摇晃。在玩游戏时眩晕的玩家可以禁用此功能以缓解症状。",
+	"sodium.options.attack_indicator.tooltip": "控制攻击指示器在屏幕上的显示位置。",
+	"sodium.options.autosave_indicator.tooltip": "如果启用，当游戏将世界保存到硬盘时，将显示一个提示。",
+	"sodium.options.graphics_quality.tooltip": "默认图形质量控制一些原版选项，并且是模组兼容性所必需的。如果以下选项设置为“默认”，则它们将使用此处的设置。",
+	"sodium.options.clouds_quality.tooltip": "控制云的渲染形式。",
+	"sodium.options.weather_quality.tooltip": "控制雨雪效果的渲染质量。",
+	"sodium.options.leaves_quality.name": "树叶",
+	"sodium.options.leaves_quality.tooltip": "控制树叶的渲染质量。",
+	"sodium.options.particle_quality.tooltip": "控制同一时刻屏幕上可以出现的最大粒子数。",
+	"sodium.options.smooth_lighting.tooltip": "控制方块是否被平滑地照亮和着色。这会略微增加重新加载区块所需的时间，但不会影响帧率。",
+	"sodium.options.biome_blend.value": "%s方块",
+	"sodium.options.biome_blend.tooltip": "控制对生物群落进行采样以进行方块着色的范围。较高的值会大大增加加载区块所需的时间，但画质提升会更少。",
+	"sodium.options.entity_distance.tooltip": "控制实体可以渲染的最远距离。越高的值渲染距离越远，但会降低帧率。",
+	"sodium.options.entity_shadows.tooltip": "如果启用，将在生物和其他实体下方渲染简单的阴影。",
+	"sodium.options.vignette.name": "晕影",
+	"sodium.options.vignette.tooltip": "如果启用，将在玩家的视图上呈现晕影效果。除非填充率受到限制，否则不太可能对帧率产生影响。",
+	"sodium.options.mipmap_levels.tooltip": "控制用于方块模型纹理的MipMap的数量。较高的值可以更好地呈现远处的方块，但可能会对许多纹理动画的性能产生不利影响。",
```

```suggestion
	"sodium.options.mipmap_levels.tooltip": "控制用于方块模型纹理的Mipmap的数量。较高的值可以更好地呈现远处的方块，但可能会对许多纹理动画的性能产生不利影响。",
```
一般来说mipmap纯小写
大写也仅大写首字母